packages/nextjs-kit/src/components/contentWithImage.tsx

```diff
@@ -61,7 +61,7 @@ const ContentWithImage: React.FC<ContentProps> = ({
 			</div>
 
 			<div
-				className="ps-break-words ps-mt-12"
+				className="ps-break-words ps-mt-12 ps-wp-content"
```

Comment:
It's a bit strange to use a WordPress specific class in our `nextjs-kit`, as this component is also used in the Drupal starter.

Reply:
I guess I'm not certain the class should go on this component, but instead should go on the implementation in the WordPress based starters, because this class won't do anything for the Drupal starter, right?

Reply:
Should do I add a class name prop for it?

Reply:
I think instead of adding the class to the component in `nextjs-kit` here, we should add it to the implementation of the components in the `next-wordpress-starter` so that the class doesn't show up in the Drupal implementations.